### ISSUE TITLE
[Bug]: FTB-Library triggers Architectury crash

### ISSUE THREAD

**Issue by revdave33** [FTBTeam/FTB-Mods-Issues#406]
### Mod

FTB Library

### Mod version

ftb-library-forge-1900.3.8-build-135

### Forge / Fabric version

Forge 1.19.2 43.0.2

### What issue are you having?

I activate ftb-library (in anticipation of using ftb-ultimine) it triggers a crash with Architectury 6.1.39-forge. When I disable ftb-library the game runs fine.

### Crashlogs

https://paste.feed-the-beast.com/view/555e6c24

### Steps to reproduce

Add ftb-library to mod directory containing architectury.
start minecraft

**Comment by Raidobw2** [comment]
FTB Library and FTB Ultimine aren't ported to 1.19.1 and 1.19.2, while Architectury is.

FTB Library will need to update its code to work with the new version of Architectury, right now the old code is trying to access classes that no longer exist and it's causing that load crash.

On the CurseForge page you can see this under versions:

This build of FTB Library is only for 1.19, not guaranteed to work on 1.19.1 or 1.19.2 yet (update is being held a bit to further versions):
![image](https://user-images.githubusercontent.com/17010632/183778430-5a5f57b9-3d27-4180-b8a6-fd1c085c64fb.png)

This build of Architectury specifically supports 1.19.1 and 1.19.2:
![image](https://user-images.githubusercontent.com/17010632/183778476-e18e0137-2aaa-4a52-9aed-fb9e8473fa84.png)

In short, Ultimine is not yet available on 1.19.1 nor 1.19.2, you will need to wait until it shows in the CurseForge versions
